cmd/cloudshell_open/appfile.go

```diff
@@ -33,8 +33,9 @@ type env struct {
 }
 
 type appFile struct {
-	Name string         `json:"name"`
-	Env  map[string]env `json:"env"`
+	Name        string         `json:"name"`
+	Env         map[string]env `json:"env"`
+	RequireAuth *bool          `json:"require-auth"`
```

Comment:
You can drop `*` and just make it false-by-default.

Since we don't serialize this object, having the extra field listed doesn't make a difference.

It would be useful to have `*bool` when/if you're trying to distinguish `nil` (unset) vs `false` (zero value).